pytripgui/model/plot_model.py

```diff
@@ -1,47 +1,85 @@
+import logging
+
 from pytripgui.model.dos import Dos
 from pytripgui.model.let import Let
 from pytripgui.model.ctx import Ctx
 
-import pytrip as pt
-
-import logging
 logger = logging.getLogger(__name__)
 
 
 class ProjectionSelector:
     def __init__(self):
-        self.current_x_slice = 50
-        self.current_y_slice = 125
-        self.current_z_slice = 125
+        self.current_T_slice = 0
+        self.current_S_slice = 0
+        self.current_C_slice = 0
+
+        self.transversal_depth = 0
+        self.sagittal_depth = 0
+        self.coronal_depth = 0
```

Comment:
can you add some explanation what is difference between slice and depth? (And units, if applicable)

Reply:
if You have a cube, and take one value for x, and all y and z (for given x), then x is depth, but i named it transversal instead of x_depth

Reply:
I changed names to more appropriate